client/createHeaders.go

```diff
@@ -6,18 +6,7 @@ import (
 	"strconv"
 )
 
-func headerResponse(contentType string, contentLength int, pathLocation string, fileName string, fileExtension string) string {
-	// Construct the request string
-	requestString := fmt.Sprintf(`POST SFTP 1.0
-ContentType: %s
-ContentLength: %d
-Path: %s
-GUID: 0dada1dc-cb0a-463a-b028-7d04a8a5d3e4
-FileName: %s
-FileSystem: Unix
-FileExtension: %s
-Authorization: null`, contentType, contentLength, pathLocation, fileName, fileExtension)
-
+func createHeaders(requestString string) string {
 	// Convert the request string to a byte array
 	byteArray := []byte(requestString)
```

Comment:
Is het mogelijk om een []byte mee te geven met de functie in plaats van de convert te doen?

Reply:
Ik vermoed van wel, moet ik even naar kijken of dat kan.

Reply:
Voor nu is het geen probleem, alleen misschien goed om later naar te kijken.